wp-content/plugins/dwt_listing_framework/redux-extensions/extensions/wbc_importer/demo-data/RTL/theme-options.txt
{"last_tab":"","dwt_listing_header-layout":"2","dwt_listing_topbar":"1","dwt_listing_slogan":"Welcome To DWT Listing Theme","dwt_listing_locationz":"0","dwt_listing_share":{"Facebook":"Facebook","Twitter":"Twitter","Skype":"Skype","LinkedIn":"LinkedIn","Youtube":"Youtube","GooglePlus":"GooglePlus"},"dwt_listing_site-spinner":"1","dwt_listing_spinner-logo":{"url":"https:\/\/demo-content.downtown-directory.com\/rtl-version\/wp-content\/themes\/dwt-listing\/assets\/images\/loader.gif","id":"","height":"","width":"","thumbnail":"","title":"","caption":"","alt":"","description":""},"dwt_listing_spinner-text":"\u0627\u0631\u062c\u0648\u0643 \u0627\u0646\u062a\u0638\u0631","dwt_listing_logo":{"url":"https:\/\/demo-content.downtown-directory.com\/rtl-version\/wp-content\/uploads\/sites\/2\/2018\/06\/logo.png","id":"3279","height":"35","width":"212","thumbnail":"https:\/\/demo-content.downtown-directory.com\/rtl-version\/wp-content\/uploads\/sites\/2\/2018\/06\/logo-150x35.png","title":"logo","caption":"","alt":"","description":""},"dwt_listing_logo-transparent":{"url":"https:\/\/demo-content.downtown-directory.com\/rtl-version\/wp-content\/themes\/dwt-listing\/assets\/images\/logo-white.png","id":"","height":"","width":"","thumbnail":"","title":"","caption":"","alt":"","description":""},"dwt_listing_sticky-header":"","dwt_listing_header-btn":"1","dwt_listing_header-text":"\u0625\u0636\u0627\u0641\u0629 \u0642\u0627\u0626\u0645\u0629","dwt_listing_header-bg":{"background-color":"","background-repeat":"no-repeat","background-size":"cover","background-attachment":"fixed","background-position":"center center","background-image":"https:\/\/demo-content.downtown-directory.com\/rtl-version\/wp-content\/themes\/dwt-listing\/assets\/images\/collage.jpg","media":{"id":"","height":"","width":"","thumbnail":""}},"dwt_listing_header-page":"2528","dwt_listing_faqs-page":"3082","dwt_listing_disable_edit":"0","dwt_listing_disable_submission":"1","dwt_listing_disable_header_text":"View All Listings","dwt_listing_disable_after_page":"","typography-body":{"font-family":"Poppins","font-options":"","google":"1","font-weight":"400","font-style":"","subsets":"","font-size":"14px","line-height":"26px","color":"#444"},"dwt_nav-typo":{"font-family":"Poppins","font-options":"","google":"1","font-weight":"400","font-style":"","subsets":"","font-size":"14px","line-height":"50px","color":"#000000"},"dwt_h2_typo":{"font-family":"Poppins","font-options":"","google":"1","font-weight":"500","font-style":"","subsets":"","font-size":"30px","line-height":"35px","color":"#444"},"dwt_h3_typo":{"font-family":"Poppins","font-options":"","google":"1","font-weight":"400","font-style":"","subsets":"","font-size":"20px","line-height":"20px","color":"#444"},"dwt_h4_typo":{"font-family":"Poppins","font-options":"","google":"1","font-weight":"400","font-style":"","subsets":"","font-size":"18px","line-height":"18px","color":"#444"},"dwt_h5_typo":{"font-family":"Poppins","font-options":"","google":"1","font-weight":"400","font-style":"","subsets":"","font-size":"16px","line-height":"22px","color":"#444"},"dwt_h6_typo":{"font-family":"Poppins","font-options":"","google":"1","font-weight":"400","font-style":"","subsets":"","font-size":"14px","line-height":"18px","color":"#444"},"dwt_p_typo":{"font-family":"Open Sans","font-options":"","google":"1","font-weight":"400","font-style":"","subsets":"","font-size":"16px","line-height":"30px","color":"#999"},"dwt_primary_clr":"#e52d27","dwt_btnz_plate":{"regular":"#242424","hover":"#e52d27","active":"#ea5652"},"dwt_naviagtion_color":{"alpha":"1"},"dwt_naviagtion_solid_color":{"color":"#fff","alpha":"1","rgba":"rgba(255,255,255,1)"},"dwt_breadcrumnb_color":{"alpha":".8"},"dwt_topbar_clr":"#503263","dwt_listing_css_editor":"#youclass{\r\nmargin: 0 auto;\r\n}","dwt_listing_js_editor":"jQuery(document).ready(function(){\r\n\r\n});","dwt_listing_enable_packages":"1","dwt_listing_package_type":"2549","dwt_listing_disable_menu":"1","dwt_listing_brand_default_image":{"url":"https:\/\/demo-content.downtown-directory.com\/rtl-version\/wp-content\/themes\/dwt-listing\/assets\/images\/brand.png","id":"","height":"","width":"","thumbnail":"","title":"","caption":"","alt":"","description":""},"dwt_listing_new_user_email_verification":"0","admin_contact_page":"","dwt_listing_new_user_email_to_admin":"1","dwt_listing_new_user_email_to_user":"1","dwt_listing_profile-dashboard":"1","dwt_listing_profile-page":"2530","dwt_listing_user-default-image":{"url":"https:\/\/demo-content.downtown-directory.com\/rtl-version\/wp-content\/themes\/dwt-listing\/assets\/images\/users\/defualt.jpg","id":"","height":"","width":"","thumbnail":"","title":"","caption":"","alt":"","description":""},"users_per_page":"12","dwt_listing_title_limit":"45","wo_pack_approve":"1","dwt_listing_allow_loc":"1","dwt_listing_gmap_lang":"en","dwt_listing_show_street_view":"1","dwt_listing_ad_approval":"auto","dwt_listing_up_approval":"auto","dwt_listing_email_on_listing":"1","dwt_listing_get_listing_email":"scriptsbundle@hotmail.com","email_on_ad_approval":"1","enable_report_option":"1","report_options":"Spam|Offensive|Duplicated|Fake","report_limit":"50","report_action":"2","report_email":"scriptsbundle@hotmail.com","dwt_listing_is_claim":"1","dwt_listing_is_admin_email":"1","dwt_listing_defualt_listing_image":{"url":"https:\/\/demo-content.downtown-directory.com\/rtl-version\/wp-content\/themes\/dwt-listing\/assets\/images\/no-image.png","id":"","height":"","width":"","thumbnail":"","title":"","caption":"","alt":"","description":""},"dwt_listing_standard_images_size":"1","dwt_listing_image_up_size":"20971520-20MB","dwt_listing_allow_country_location":"1","dwt_listing_bad_words_filter":"","dwt_listing_bad_words_replace":"","dwt_listing_coupon_admin_note":"1","dwt_listing_coupon_admin_note_desc":"","dwt_listing_coupon_desc_limit":"170","dwt_listing_list_title":"\u0639\u0646\u0648\u0627\u0646 \u0627\u0644\u0642\u064a\u062f","dwt_listing_list_tool":"\u0636\u0639 \u0627\u0633\u0645 \u0634\u0631\u0643\u062a\u0643 \u0648\u0627\u0633\u0645\u062d \u0644\u0644\u0639\u0627\u0644\u0645 \u0628\u0645\u0639\u0631\u0641\u0629","dwt_listing_list_place":"\u0639\u0644\u0649 \u0633\u0628\u064a\u0644 \u0627\u0644\u0645\u062b\u0627\u0644 \u0627\u0644\u0637\u0639\u0627\u0645 \u0627\u0644\u0642\u0627\u0631\u064a","dwt_listing_list_tagline":"\u0639\u0646\u0648\u0627\u0646","dwt_listing_list_tagline_tool":"\u062f\u0639 \u0627\u0644\u0646\u0627\u0633 \u064a\u0639\u0631\u0641\u0648\u0646 \u0645\u0627 \u062a\u062a\u062e\u0635\u0635 \u0641\u064a\u0647","dwt_listing_list_tagline_place":"\u0645\u062b\u0644\u0627 \u0631\u0626\u064a\u0633 \u0627\u0644\u0637\u0639\u0627\u0645 \u0627\u0644\u0642\u0627\u0631\u064a","dwt_listing_list_category":"\u0627\u062e\u062a\u0631 \u0627\u0644\u0641\u0626\u0629","dwt_listing_list_contact":"\u0631\u0642\u0645 \u0627\u0644\u0647\u0627\u062a\u0641","dwt_listing_list_contact_tool":"\u0631\u0642\u0645 \u0627\u0644\u0627\u062a\u0635\u0627\u0644 \u0642\u0627\u0626\u0645\u0629","dwt_listing_list_contact_place":"+99 3331 234567","dwt_listing_list_web":"\u0631\u0627\u0628\u0637 \u0627\u0644\u0645\u0648\u0642\u0639","dwt_listing_list_web_tool":"\u0627\u0643\u062a\u0628 \u0639\u0646\u0648\u0627\u0646 \u0645\u0648\u0642\u0639\u0643 \u0639\u0644\u0649 \u0627\u0644\u0648\u064a\u0628","dwt_listing_list_web_place":"\u0648\u0635\u0644\u062a\u0643","dwt_listing_list_pricetype":"\u062d\u062f\u062f \u0646\u0648\u0639 \u0627\u0644\u0633\u0639\u0631","dwt_listing_list_currenct":"\u0639\u0645\u0644\u0629 \u0627\u0644\u0625\u062f\u0631\u0627\u062c","dwt_listing_list_price_from":"\u0627\u0644\u0633\u0639\u0631 \u064a\u0628\u062f\u0623 \u0645\u0646","dwt_listing_list_price_to":"\u0627\u0644\u0633\u0639\u0631 \u0627\u0644\u0649","dwt_listing_b_h_section":"\u0627\u062e\u062a\u0631 \u0633\u0627\u0639\u0627\u062a \u0639\u0645\u0644\u0643","dwt_listing_b_h_time":"\u0627\u062e\u062a\u0631 \u0645\u0646\u0637\u0642\u062a\u0643 \u0627\u0644\u0632\u0645\u0646\u064a\u0629","dwt_listing_list_video":"\u0631\u0627\u0628\u0637 \u0627\u0644\u0641\u064a\u062f\u064a\u0648","dwt_listing_list_video_tool":"\u062a\u0642\u062f\u064a\u0645 \u0631\u0627\u0628\u0637 \u0641\u064a\u062f\u064a\u0648 \u064a\u0648\u062a\u064a\u0648\u0628 \u062a\u0645\u0647\u064a\u062f\u064a\u0629","dwt_listing_list_video_place":"\u0631\u0627\u0628\u0637 \u0627\u0644\u0641\u064a\u062f\u064a\u0648","dwt_listing_list_tags":"\u0642\u0627\u0626\u0645\u0629 \u0627\u0644\u0639\u0644\u0627\u0645\u0627\u062a \/ \u0627\u0644\u0643\u0644\u0645\u0627\u062a \u0627\u0644\u0631\u0626\u064a\u0633\u064a\u0629","dwt_listing_list_tags_place":"\u062a\u0642\u062f\u064a\u0645 \u0639\u0644\u0627\u0645\u0627\u062a\u0643 (\u0645\u0641\u0635\u0648\u0644\u0629 \u0628\u0641\u0648\u0627\u0635\u0644)","dwt_listing_coupon_title":"\u0642\u0633\u064a\u0645\u0629 \u062a\u0631\u0648\u064a\u062c\u064a\u0629","dwt_listing_coupon_code":"\u0643\u0648\u062f \u0627\u0644\u0642\u0633\u064a\u0645\u0629","dwt_listing_coupon_referral":"\u0631\u0627\u0628\u0637 \u0627\u0644\u0625\u062d\u0627\u0644\u0629 \u0627\u0644\u062e\u0627\u0635 \u0628\u0643","dwt_listing_coupon_expiry_date":"\u062a\u0627\u0631\u064a\u062e \u0627\u0646\u062a\u0647\u0627\u0621 \u0627\u0644\u0642\u0633\u064a\u0645\u0629","dwt_listing_coupon_desc":"\u0627\u0644\u0643\u0648\u0628\u0648\u0646 \u0648\u0635\u0641 \u0642\u0635\u064a\u0631","dwt_listing_list_google_loc":"\u0642\u0627\u0626\u0645\u0629 \u0627\u0644\u0645\u0648\u0642\u0639","dwt_listing_list_google_loc_tool":"\u0645\u0648\u0642\u0639 \u0646\u0634\u0627\u0637\u0643 \u0627\u0644\u062a\u062c\u0627\u0631\u064a \u0627\u0644\u0641\u0639\u0644\u064a \u060c \u0641\u0633\u064a\u0638\u0647\u0631 \u0641\u064a \u0646\u062a\u064a\u062c\u0629 \u0627\u0644\u0628\u062d\u062b","dwt_listing_list_google_loc_place":"Deluxe Diner Division No. 18\u060c Canada","dwt_listing_list_lati":"\u062e\u0637 \u0627\u0644\u0639\u0631\u0636","dwt_listing_list_lati_tool":"\u0633\u064a\u062a\u0645 \u0645\u0644\u0621 \u0627\u0644\u0625\u062d\u062f\u0627\u062b\u064a\u0627\u062a \u062a\u0644\u0642\u0627\u0626\u064a\u064b\u0627 \u0625\u0630\u0627 \u0642\u0645\u062a \u0628\u062a\u0648\u0641\u064a\u0631 \u0645\u0648\u0642\u0639 \u0627\u0644\u0639\u0645\u0644","dwt_listing_list_lati_place":"56.1304","dwt_listing_list_longi":"\u062e\u0637 \u0627\u0644\u0637\u0648\u0644","dwt_listing_list_longi_tool":"\u0633\u064a\u062a\u0645 \u0645\u0644\u0621 \u0627\u0644\u0625\u062d\u062f\u0627\u062b\u064a\u0627\u062a \u062a\u0644\u0642\u0627\u0626\u064a\u064b\u0627 \u0625\u0630\u0627 \u0642\u0645\u062a \u0628\u062a\u0648\u0641\u064a\u0631 \u0645\u0648\u0642\u0639 \u0627\u0644\u0639\u0645\u0644","dwt_listing_list_longi_place":"106.3468","dwt_listing_b_title":"\u0627\u0633\u0645 \u0627\u0644\u0639\u0644\u0627\u0645\u0629 \u0627\u0644\u062a\u062c\u0627\u0631\u064a\u0629","dwt_listing_b_placetitle":"\u0645\u0637\u0639\u0645 KnK & \u0627\u0644\u0648\u062c\u0628\u0627\u062a \u0627\u0644\u0633\u0631\u064a\u0639\u0629","dwt_listing_brand_name_logo":"\u0634\u0639\u0627\u0631 \u0627\u0644\u0639\u0644\u0627\u0645\u0629 \u0627\u0644\u062a\u062c\u0627\u0631\u064a\u0629","dwt_listing_list_desc":"\u0648\u0635\u0641","dwt_listing_list_gallery":"\u0645\u0639\u0631\u0636 \u0627\u0644\u0642\u0627\u0626\u0645\u0629","dwt_listing_list_gallery_desc":"\u0623\u0641\u0644\u062a \u0627\u0644\u0645\u0644\u0641\u0627\u062a \u0647\u0646\u0627 \u0623\u0648 \u0627\u0646\u0642\u0631 \u0644\u0644\u062a\u062d\u0645\u064a\u0644","dwt_listing_list_socail_media":"\u0639\u0646\u0627\u0648\u064a\u0646 \u0627\u0644\u062a\u0648\u0627\u0635\u0644 \u0627\u0644\u0627\u062c\u062a\u0645\u0627\u0639\u064a","dwt_listing_list_form_btn":"\u0645\u0639\u0627\u064a\u0646\u0629 \u0648\u0625\u0631\u0633\u0627\u0644","sb_location_titles":"\u0627\u0644\u062f\u0648\u0644\u0629 | \u0627\u0644\u062f\u0648\u0644\u0629 | \u0627\u0644\u0645\u062f\u064a\u0646\u0629 | \u062a\u0627\u0648\u0646","dwt_listing_list_cover":"Cover Image","dwt_listing_list_cover_reco":"Recommended cover size is 1920x550 px","dwt_listing_seacrh_page":"3291","dwt_listing_seacrh_layout":"map","dwt_listing_enable_mapfilters":"1","donwtown_search_bytitle":"1","donwtown_search_byloc":"1","donwtown_search_bcats":"1","donwtown_search_byprice":"1","donwtown_search_byopen":"1","donwtown_search_byrated":"1","donwtown_search_byregion":"1","donwtown_search_bytags":"1","donwtown_search_byfeatures":"0","donwtown_search_bysort":"1","dwt_listing_search_layout_style":"grid2","listing_title_limt":"1","grid_title_limit":"20","dwt_listing_sidebar_position":"right","dwt_listing_enable_video_option":"1","dwt_listing_video_icon":{"url":"https:\/\/demo-content.downtown-directory.com\/rtl-version\/wp-content\/themes\/dwt-listing\/assets\/images\/play-button.png","id":"","height":"","width":"","thumbnail":"","title":"","caption":"","alt":"","description":""},"feature_or_simple":"1","feature_on_search":"0","dwt_listing_feature_on_search":"grid2","max_ads_feature":"5","feature_ads_title":"Featured Ads","search_ad_720_1":"<img class=\"img-responsive\" alt=\"not found\" src=\"https:\/\/demo-content.downtown-directory.com\/rtl-version\/wp-content\/themes\/dwt-listing\/assets\/images\/eds\/720x120-1.jpg\"> ","search_ad_720_2":"<img class=\"img-responsive\" alt=\"not found\" src=\"https:\/\/demo-content.downtown-directory.com\/rtl-version\/wp-content\/themes\/dwt-listing\/assets\/images\/eds\/720x90-2.png\"> ","dwt_listing_lp_style":"elegent","dwt_listing_cover_default_image":{"url":"https:\/\/demo-content.downtown-directory.com\/rtl-version\/wp-content\/themes\/dwt-listing\/assets\/images\/listing-cover.png","id":"","height":"","width":"","thumbnail":"","title":"","caption":"","alt":"","description":""},"dwt_listing_view-layout-manager-minimal":{"enabled":{"placebo":"placebo","ad_slot_1":"Ad Slot 1","desc":"Description","menu":"Menu","listing_features":"Features","location":"Location","form_fields":"Custom Fields","video":"Video","ad_slot_2":"Ad Slot 2","reviews":"Reviews"},"disabled":{"placebo":"placebo"}},"dwt_listing_view-layout-manager":{"enabled":{"placebo":"placebo","slider":"Slider","ad_slot_1":"Ad Slot 1","desc":"Description","listing_features":"Features","location":"Location","form_fields":"Custom Fields","video":"Video","ad_slot_2":"Ad Slot 2","reviews":"Reviews"},"disabled":{"placebo":"placebo","menu":"Menu"}},"dwt_listing_elegent_manager":{"enabled":{"placebo":"placebo","ad_slot_1":"Ad Slot 1","desc":"Description","menu":"Menu","listing_features":"Features","location":"Location","form_fields":"Custom Fields","video":"Video","ad_slot_2":"Ad Slot 2","reviews":"Reviews"},"disabled":{"placebo":"placebo"}},"dwt_listing_menu_type_col":"column_1","dwt_listing_slot_1":"<img class=\"img-responsive center-block\" alt=\"not found\" src=\"https:\/\/demo-content.downtown-directory.com\/rtl-version\/wp-content\/themes\/dwt-listing\/assets\/images\/eds\/720x120-1.jpg\"> ","dwt_listing_slot_2":"<img class=\"img-responsive center-block alt=\"not found\" src=\"https:\/\/demo-content.downtown-directory.com\/rtl-version\/wp-content\/themes\/dwt-listing\/assets\/images\/eds\/720x90-2.png\"> ","dwt_listing_sidebar-layout-manager":{"enabled":{"placebo":"placebo","card":"Profile Card","coupon":"Coupon","events":"Related Event","hours":"Business Hours","claim":"Claim","pricing":"Pricing","tags":"Tags"},"disabled":{"placebo":"placebo"}},"dwt_listing_timezone_txt":"\u0642\u0627\u0626\u0645\u0629 \u0627\u0644\u0645\u0646\u0637\u0642\u0629 \u0627\u0644\u0632\u0645\u0646\u064a\u0629","dl_desc":"\u0648\u0635\u0641","dl_menu":"\u0642\u0627\u0626\u0645\u0629 \u0637\u0639\u0627\u0645","dl_advert":"\u0627\u0644\u0625\u0639\u0644\u0627\u0646\u0627\u062a","dl_amenties":"\u0648\u0633\u0627\u0626\u0644 \u0627\u0644\u0631\u0627\u062d\u0629","dl_custom":"\u062a\u0641\u0627\u0635\u064a\u0644 \u0627\u0636\u0627\u0641\u064a\u0629","dl_location":"\u0645\u0648\u0642\u0639\u0643","dl_video":"\u0641\u064a\u062f\u064a\u0648","dl_reviews":"\u0627\u0644\u062a\u0639\u0644\u064a\u0642\u0627\u062a","dwt_listing_form-layout-manager":{"enabled":{"placebo":"placebo","title_cat":"Title & Category Section","price_type":"Price Type Section","buiness_hours":"Business Hours Section","social_links":"Social Media Section","desc_sec":"Description & Gallery","coupon":"Coupon Section","location":"Location Section"}},"dwt_listing_enable_leads":"1","dwt_leads_per_page":"20","dwt_listing_show_pkg":"1","dwt_listing_packages":"2532","d_p_exp":"\u0627\u0646\u062a\u0647\u0627\u0621 \u0627\u0644\u062d\u0632\u0645\u0629","d_reg_listing":"\u0642\u0627\u0626\u0645\u0629 \u0627\u0644\u0639\u0627\u062f\u064a\u0629","d_feat_listing":"\u0642\u0627\u0626\u0645\u0629 \u0645\u0645\u064a\u0632\u0629","d_feat_for":"\u0645\u0645\u064a\u0632\u0629 \u0644","d_vid_listing":"\u0642\u0627\u0626\u0645\u0629 \u0627\u0644\u0641\u064a\u062f\u064a\u0648","d_web_link":"\u0631\u0627\u0628\u0637 \u0627\u0644\u0645\u0648\u0642\u0639","d_no_images":"\u0639\u062f\u062f \u0627\u0644\u0635\u0648\u0631","d_p_range":"\u0646\u0637\u0627\u0642 \u0627\u0644\u0633\u0639\u0631","d_b_hours":"\u0633\u0627\u0639\u0627\u062a \u0627\u0644\u0639\u0645\u0644","d_llow_tag":"\u0627\u0644\u0633\u0645\u0627\u062d \u0628\u0627\u0644\u0639\u0644\u0627\u0645\u0627\u062a","d_bump_listing":"\u0642\u0627\u0626\u0645\u0629 \u0627\u0644\u0627\u0647\u062a\u0632\u0627\u0632","d_coupon_code":"\u0627\u0644\u0633\u0645\u0627\u062d \u0643\u0648\u062f \u0627\u0644\u0642\u0633\u064a\u0645\u0629","d_create_event":"\u0627\u0646\u0634\u0627\u0621 \u062d\u062f\u062b","d_pkg_daytxt":"\u0623\u064a\u0627\u0645","d_pkg_unlimited":"\u063a\u064a\u0631 \u0645\u062d\u062f\u0648\u062f","d_never_exp":"\u0644\u0627 \u062a\u0646\u062a\u0647\u064a","d_pkg_yes":"\u0646\u0639\u0645 \u0641\u0639\u0644\u0627","d_pkg_no":"\u0644\u0627","dwt_listing_blog-post-title":"\u0622\u062e\u0631 \u0627\u0644\u0623\u062e\u0628\u0627\u0631 \u0648\u0627\u0644\u0627\u062a\u062c\u0627\u0647\u0627\u062a","dwt_listing_blog-layout":{"enabled":{"placebo":"placebo","content":"Content Area ","sidebar":"Sidebar"}},"dwt_listing_blog-singlelayout":{"enabled":{"placebo":"placebo","singlepost":"Post Detail ","singlesidebar":"Sidebar"}},"dwt_listing_review_send_email":"1","dwt_listing_review_reply_email":"1","dwt_listing_review_rating_limit":"1","dwt_listing_review_permission":"0","dwt_listing_review_enable_stars":"1","dwt_listing_review_enable_gallery":"0","dwt_listing_review_enable_emoji":"1","dwt_listing_review_upload_limit":"5","dwt_listing_review_images_size":"2097152-2MB","review_limit_listing_page":"5","dwt_listing_allreview_page":"2728","dwt_listing_review_limit_btn_text":"\u0639\u0631\u0636 \u062c\u0645\u064a\u0639 \u0627\u0644\u0627\u0633\u062a\u0639\u0631\u0627\u0636\u0627\u062a","dwt_listing_review_all_pagination_limit":"8","dwt_listing_enable_names":"1","dwt_listing_review_taglines_limit":"20|40|60|80","dwt_listing_review_taglines_titles":"\u0648\u0633\u062c\u0644 \u0647\u0627\u0645\u064a\u0644\u062a\u0648\u0646 | \u0627\u0644\u0647\u0648\u0627\u0629 | \u0627\u0644\u0646\u062e\u0628\u0629 | \u0627\u0644\u0641\u0646\u064a\u0629","dwt_listing_show_total_ratings":"1","dwt_listing_cat_slug":"categories","dwt_listing_tags_slug":"tags","dwt_listing_loc_slug":"location","dwt_listing_listing_slug":"listing","dwt_event_cat_slug":"event-categories","dwt_listing_event_slug":"events","related_products":"1","max_related_products":"10","related_products_title":"\u0645\u0646\u062a\u062c\u0627\u062a \u0630\u0627\u062a \u0635\u0644\u0647","show_banners_on_product":"1","opt-slides-do":[{"title":"","url":"#","sort":"0","attachment_id":"3075","thumb":"https:\/\/demo-content.downtown-directory.com\/rtl-version\/wp-content\/uploads\/sites\/2\/2018\/02\/Untitled-8-150x150.png","image":"https:\/\/demo-content.downtown-directory.com\/rtl-version\/wp-content\/uploads\/sites\/2\/2018\/02\/Untitled-8.png","height":"250","width":"887"},{"title":"","url":"#","sort":"1","attachment_id":"3074","thumb":"https:\/\/demo-content.downtown-directory.com\/rtl-version\/wp-content\/uploads\/sites\/2\/2018\/02\/Untitled-8-150x150.jpg","image":"https:\/\/demo-content.downtown-directory.com\/rtl-version\/wp-content\/uploads\/sites\/2\/2018\/02\/Untitled-8.jpg","height":"250","width":"870"}],"dwt_listing_recaptcha":"","google_api_key":"","google_api_secret":"","mailchimp_api_key":"","mailchimp_list_id":"","fb_api_key":"","gmail_api_key":"","redirect_uri":"","dwt_listing_enable_geo":"1","dwt_geo_api_settings":"geo_ip","dwt_listing_enable_map":"1","dwt_map_selection":"open_street","gmap_api_key":"","dwt_listing_default_lat":"40.7127837","dwt_listing_default_long":"-74.00594130000002","dwt_listing_event_page":"2830","dwt_listing_event_layout":"topbar","dwt_listing_event_send_email":"1","dwt_listing_event_send_email_admin":"1","dwt_listing_event_admin_email":"scriptsbundle@hotmail.com","dwt_listing_event_approval":"1","dwt_listing_event_up_approval":"auto","dwt_listing_event_upload_limit":"5","dwt_listing_event_images_size":"20971520-20MB","dwt_listing_defualt_event_image":{"url":"https:\/\/demo-content.downtown-directory.com\/rtl-version\/wp-content\/themes\/dwt-listing\/assets\/images\/event.jpg","id":"","height":"","width":"","thumbnail":"","title":"","caption":"","alt":"","description":""},"dwt_listing_events-filter-manager":{"enabled":{"placebo":"placebo","by_title":"By Title","by_category":"By Categories","by_location":"By Location"},"disabled":{"placebo":"placebo"}},"dwt_listing_msg_subject_on_new_ad":"You have new Listing - DWT Listing","dwt_listing_msg_from_on_new_ad":"From: Down Town Listing Directory Theme RTL Version <scriptsbundle@hotmail.com>","dwt_listing_msg_on_new_ad":"<table class=\"body\" style=\"border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; background-color: #f6f6f6; width: 100%;\" border=\"0\" cellspacing=\"0\" cellpadding=\"0\">\r\n<tbody>\r\n<tr>\r\n<td style=\"font-family: sans-serif; font-size: 14px; vertical-align: top;\"><\/td>\r\n<td class=\"container\" style=\"font-family: sans-serif; font-size: 14px; vertical-align: top; display: block; max-width: 580px; padding: 10px; width: 580px; margin: 0 auto !important;\">\r\n<div class=\"content\" style=\"box-sizing: border-box; display: block; margin: 0 auto; max-width: 580px; padding: 10px;\">\r\n<table class=\"main\" style=\"border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; background: #fff; border-radius: 3px; width: 100%;\">\r\n<tbody>\r\n<tr>\r\n<td class=\"wrapper\" style=\"font-family: sans-serif; font-size: 14px; vertical-align: top; box-sizing: border-box; padding: 20px;\">\r\n<table style=\"border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%;\" border=\"0\" cellspacing=\"0\" cellpadding=\"0\">\r\n<tbody>\r\n<tr style=\"font-family: 'Helvetica Neue',Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; margin: 0;\">\r\n<td class=\"alert\" style=\"font-family: 'Helvetica Neue',Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 16px; vertical-align: top; color: #000; font-weight: 500; text-align: center; border-radius: 3px 3px 0 0; background-color: #fff; margin: 0; padding: 20px;\" align=\"center\" valign=\"top\" bgcolor=\"#fff\"><img class=\"alignnone size-full wp-image-1437\" src=\"https:\/\/listing.dwt_listing_directory.com\/wp-content\/uploads\/2018\/02\/logo.png\" alt=\"not found\" width=\"200\" height=\"40\" \/><br\/>\r\nA Designing and development company<\/td>\r\n<\/tr>\r\n<tr>\r\n<td style=\"font-family: sans-serif; font-size: 14px; vertical-align: top;\">\r\n<p style=\"font-family: sans-serif

wp-content/themes/dwt-listing/inc/options.php
<?php
    /**
     * ReduxFramework Sample Config File
     * For full documentation, please visit: http://docs.reduxframework.com/
     */

    if ( ! class_exists( 'Redux' ) ) {
        return;
    }


    // This is your option name where all the Redux data is stored.
    $opt_name = "dwt_listing_options";
	$sample_patterns = $sampleHTML = '';
    // This line is only for altering the demo. Can be easily removed.
    $opt_name = apply_filters( 'redux_demo/opt_name', $opt_name );

    $theme = wp_get_theme(); // For use with some settings. Not necessary.

    $args = array(
        // TYPICAL -> Change these values as you need/desire
        'opt_name'             => $opt_name,
        'display_name'         => $theme->get( 'Name' ),
        'display_version'      => $theme->get( 'Version' ),
        'menu_type'            => 'submenu',
        'allow_sub_menu'       => true,
        'menu_title'           => __( 'DWT Listing Options', 'dwt-listing' ),
        'page_title'           => __( 'DWT Listing Options', 'dwt-listing' ),
        'google_api_key'       => '',
        'google_update_weekly' => false,
        'async_typography'     => true,
        'admin_bar'            => true,
        'admin_bar_icon'       => 'dashicons-portfolio',
        'admin_bar_priority'   => 50,
        'global_variable'      => '',
        'dev_mode'             => false,
        'update_notice'        => false,
        'customizer'           => false,
        // Enable basic customizer support
        //'open_expanded'     => true,                    // Allow you to start the panel in an expanded way initially.
        //'disable_save_warn' => true,                    // Disable the save warning when a user changes a field

        // OPTIONAL -> Give you extra features
        'page_priority'        => null,
        // Order where the menu appears in the admin area. If there is any conflict, something will not show. Warning.
        'page_parent'          => 'themes.php',
        // For a full list of options, visit: http://codex.wordpress.org/Function_Reference/add_submenu_page#Parameters
        'page_permissions'     => 'manage_options',
        // Permissions needed to access the options panel.
        'menu_icon'            => '',
        // Specify a custom URL to an icon
        'last_tab'             => '',
        // Force your panel to always open to a specific tab (by id)
        'page_icon'            => 'icon-themes',
        // Icon displayed in the admin panel next to your menu_title
        'page_slug'            => '',
        // Page slug used to denote the panel, will be based off page title then menu title then opt_name if not provided
        'save_defaults'        => true,
        // On load save the defaults to DB before user clicks save or not
        'default_show'         => false,
        // If true, shows the default value next to each field that is not the default value.
        'default_mark'         => '',
        // What to print by the field's title if the value shown is default. Suggested: *
        'show_import_export'   => true,
        // Shows the Import/Export panel when not used as a field.

        // CAREFUL -> These options are for advanced use only
        'transient_time'       => 60 * MINUTE_IN_SECONDS,
        'output'               => true,
        // Global shut-off for dynamic CSS output by the framework. Will also disable google fonts output
        'output_tag'           => true,
        // Allows dynamic CSS to be generated for customizer and google fonts, but stops the dynamic CSS from going to the head
        // 'footer_credit'     => '',                   // Disable the footer credit of Redux. Please leave if you can help it.

        // FUTURE -> Not in use yet, but reserved or partially implemented. Use at your own risk.
        'database'             => '',
        // possible: options, theme_mods, theme_mods_expanded, transient. Not fully functional, warning!
        'use_cdn'              => true,
        // If you prefer not to use the CDN for Select2, Ace Editor, and others, you may download the Redux Vendor Support plugin yourself and run locally or embed it in your code.

        // HINTS
        'hints'                => array(
            'icon'          => 'el el-question-sign',
            'icon_position' => 'right',
            'icon_color'    => 'lightgray',
            'icon_size'     => 'normal',
            'tip_style'     => array(
                'color'   => 'red',
                'shadow'  => true,
                'rounded' => false,
                'style'   => '',
            ),
            'tip_position'  => array(
                'my' => 'top left',
                'at' => 'bottom right',
            ),
            'tip_effect'    => array(
                'show' => array(
                    'effect'   => 'slide',
                    'duration' => '500',
                    'event'    => 'mouseover',
                ),
                'hide' => array(
                    'effect'   => 'slide',
                    'duration' => '500',
                    'event'    => 'click mouseleave',
                ),
            ),
        )
    );

	
    Redux::setArgs( $opt_name, $args );

	/* Load Theme Functions */
	require trailingslashit( get_template_directory () ) . 'inc/admin/index.php';